package.json

```diff
@@ -1,6 +1,15 @@
 {
-  "name": "fractal-framework",
-  "version": "0.2.0",
+  "name": "@fractal-framework/fractal-framework",
+  "version": "0.0.1",
+  "main": "index.js",
```

Comment:
I *think* the existence of a `main` property is only for packages that are expected to be able to run on their own, not so much for libraries that are imported into other projects. Regardless, there is no `index.js` file in our repo here, so something needs to change.